src/applications/caregivers/components/PreSubmitInfo/SignatureInput.jsx

```diff
@@ -3,9 +3,9 @@ import ErrorableTextInput from '@department-of-veterans-affairs/formation-react/
 
 const SignatureInput = ({ fullName, required, label, setIsSigned }) => {
   const [hasError, setError] = useState(false);
-  const firstName = fullName.first.toLowerCase();
-  const lastName = fullName.last.toLowerCase();
-  const middleName = fullName.middle === undefined ? '' : fullName.middle;
+  const firstName = fullName.first ? fullName.first.toLowerCase() : '';
+  const lastName = fullName.last ? fullName.last.toLowerCase() : '';
+  const middleName = fullName.middle ? fullName.middle : '';
```

Comment:
Optional chaining could be used here!

```js
const firstName = fullName.first?.toLowerCase() || '';
```

Reply:
I was thinking about optional chaining but it just felt like more code that was not necessarily needed. Is there any benefit to this @Mottie

Reply:
Benefit? I can't think of any... is it purdy? yes 😉

Reply:
After thinking about it seems more readable so gonna go with that